src/bundle/Resources/config/services/normalizers.yaml

```diff
@@ -0,0 +1,29 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    ibexa.search.suggestion.serializer:
+        class: Symfony\Component\Serializer\Serializer
+        arguments:
+            $normalizers:
+                - '@Ibexa\Search\Serializer\Normalizer\Suggestion\ContentSuggestionNormalizer'
+                - '@Ibexa\Search\Serializer\Normalizer\Suggestion\ParentLocationNormalizer'
+            $encoders:
+                - '@serializer.encoder.json'
```

Comment:
Since `Symfony\Component\Serializer\Serializer` represents a `NormalizerInterface`, it is autoconfigured and injected into the main app's normalizer :sweat_smile:. To prevent that, we need to again disabled `autoconfigure`.

Use `symfony console debug:container --tag=serializer.normalizer` or `php bin/console debug:container --tag=serializer.normalizer` to check for incorrect tagging.

Reply:
Can confirm, I was checking this behavior not that long ago.

Reply:
The "autoconfigure" option is set to false.